pinot-tools/src/main/java/org/apache/pinot/tools/admin/command/StartServiceManagerCommand.java

```diff
@@ -192,7 +192,20 @@ public boolean execute()
   }
 
   private void startPinotService(Map<String, Object> properties) {
-    startPinotService(ServiceRole.valueOf(properties.get(PINOT_SERVICE_ROLE).toString()), properties);
+    ServiceRole role = ServiceRole.valueOf(properties.get(PINOT_SERVICE_ROLE).toString());
+    switch (role) {
+      // Broker and Server can be started in parallel always
+      case BROKER:
+      case SERVER:
+        new Thread("Starting " + role) {
+          @Override public void run() {
+            startPinotService(role, properties);
+          }
+        }.start();
+        break;
+      default:
+        startPinotService(role, properties);
```

Comment:
Add a log before starting the service and another one after the service is started and measure log the time it took to start

Reply:
I presume you mean inside `startPinotService` as it is a blocking command (then useful for both the parallel and sync paths)

Reply:
Agreed, it's good to add logs to measure the startup time.

Reply:
you want to just time each service command here right? this would be different than composite startup time

Reply:
Yes, each service. Also log the wall clock time, that will help us understand what’s debug slow starts

Reply:
ok FWIW what I used before was `-verbose:gc` for similar :P you can see time since startup for each GC event, which are often enough to figure out time like this.

```
[0.012s][info][gc] Using G1
[0.066s][info][gc] Periodic GC disabled
[0.753s][info][gc] GC(0) Pause Young (Normal) (G1 Evacuation Pause) 12M->2M(256M) 7.433ms
[1.108s][info][gc] GC(1) Pause Young (Normal) (G1 Evacuation Pause) 12M->3M(256M) 9.164ms
[1.511s][info][gc] GC(2) Pause Young (Concurrent Start) (Metadata GC Threshold) 15M->4M(256M) 6.252ms
[1.511s][info][gc] GC(3) Concurrent Cycle
[1.520s][info][gc] GC(3) Pause Remark 4M->4M(256M) 1.834ms
[1.523s][info][gc] GC(3) Pause Cleanup 4M->4M(256M) 0.172ms
[1.525s][info][gc] GC(3) Concurrent Cycle 13.819ms
```

Ex.

"starting $role at N.NNNs since launch"
"started/error starting $role at N.NNNs since launch"

Reply:
Ah nice idea. Go for it!

Reply:
Ah nice idea. Go for it!